IntentsExtension/Models/Note+Intents.swift

```diff
@@ -53,6 +53,55 @@ extension Note {
         let result = String(content[range])
         return result.droppingPrefix(Constants.titleMarkdownPrefix)
     }
+
+    private func objectFromJSONString(_ json: String) -> Any? {
+        guard let data = json.data(using: .utf8) else {
+            return nil
+        }
+
+        return try? JSONSerialization.jsonObject(with: data)
+    }
```

Comment:
This API probably fits best as a `String` extension (since it's kinda unrelated to `Note`  itself?

Reply:
done

Reply:
Nipticky!! The method is still here (unused)